src/components/CreepypastaForm.js

```diff
@@ -0,0 +1,129 @@
+import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+import styled from 'styled-components';
+import { nanoid } from 'nanoid';
+
+export default function CreepypastaFrom({ handleCreateCreepypasta }) {
+  const [wordCount, setWordCount] = useState(0);
+
+  const navigate = useNavigate();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  return (
+    <Form
+      aria-label="Create a creepypasta"
+      onSubmit={handleSubmit(creepypasta => onSubmit(creepypasta))}
+    >
+      <Field>
+        <Label htmlFor="title">Title:</Label>
+        <Input
+          {...register('title', {
+            required: 'You need a spooky title!',
+            maxLength: {
+              value: 25,
+              message: 'Calm down there! Your title is too long.',
+            },
+          })}
+          id="title"
+        />
+        <Error>{errors.title && errors.title.message}</Error>
+      </Field>
+      <Field>
+        <Label htmlFor="text">Text:</Label>
+        <Textarea
+          {...register('text', { required: 'You need a scary text!' })}
+          id="text"
+          onChange={e => setWordCount(e.target.value.split(' ').length)}
+        />
+        <Error>{errors.text && errors.text.message}</Error>
+        <Wordcount {...register('wordcount')} id="wordcount">
+          Wordcount: {wordCount}
+        </Wordcount>
+      </Field>
+      <Field>
+        <Label htmlFor="image">Image:</Label>
+        <Input
+          {...register('image', { required: 'You need a creepy image!' })}
+          id="image"
+          placeholder=" Please use an URL"
+          type="url"
+          pattern="https://"
```

Comment:
You could leave a little note for the user which format the image url needs to be.
I tried to upload an image to test it, but I failed 😂